source/includes/custom-field-selects/_update.md.erb

```diff
@@ -0,0 +1,67 @@
+## Update Custom Field Selects
+
+```http
+PATCH https://api.teamtailor.com/v1/custom-field-values/1 HTTP/1.1
```

Comment:
Is this the correct url, i would expect `custom-field-selects` here?

Reply:
you are correct.